client/yarn.lock

```diff
@@ -3052,6 +3050,27 @@
     rpc-websockets "^7.5.0"
     superstruct "^0.14.2"
 
+"@solana/web3.js@^1.92.1":
```

Comment:
We have quite many versions of `@solana/web3.js` in our lock file:

https://github.com/solana-playground/solana-playground/blob/a92a554bf03fcfafdc93b1a4502cfec54fcf3bb2/client/yarn.lock#L2970-L3053

This might require manual editing, but I think you should just be able to use the latest version for all of them to make this update work for all packages we support.